src/main/scala/diplomaticobjectmodel/model/OMISA.scala

```diff
@@ -36,3 +39,50 @@ case class OMISA(
   addressTranslationModes: Seq[OMAddressTranslationMode],
   _types: Seq[String] = Seq("OMISA", "OMCompoundType")
 ) extends OMCompoundType
+
+object OMISA {
+  // TODO the package object util has this code but is not seen in this scope
+  implicit class BooleanToAugmentedBoolean(val x: Boolean) extends AnyVal {
+    // this one's snagged from scalaz
+    def option[T](z: => T): Option[T] = if (x) Some(z) else None
+  }
+
+  def rocketISA(coreParams: RocketCoreParams, xLen: Int): OMISA = {
+    val baseInstructionSet = xLen match {
+      case 32 => if (XLen == 32) RV32E else RV32I
```

Comment:
Rocket *does not* currently implement RV32E, and therefore this should never return RV32E.

Reply:
fixed